### DISCUSSION TITLE
system_Anaiysis

### DISCUSSION THREAD

**Discussion by Bad-ri** [LUDOOS/LUDOSDOCS#9] [Q&A] [answered]
System Analysis

This section describes the final outline of our software project by detect the software methods and chosen framework.
From analyzing all the previously gathered data, we finally found the following:
Summary:  
Framework:
Methods:  
Constrains:
System Components:
Interview & persona example:
2 . 4 . System Models

Schematic Models:
Software Flowchart:
 User flow diagram:

**Comment by moha-b** [comment, marked as answer]
no interview
